Src/LinearSolvers/MLMG/AMReX_MLEBNodeFDLap_K.H

```diff
@@ -26,9 +26,19 @@ void mlebndfdlap_grad_x_doit (int i, int j, int k, Array4<Real> const& px,
     } else if (dmsk(i,j,k) < 0 && dmsk(i+1,j,k) < 0) {
         px(i,j,k) = Real(0.0);
     } else if (dmsk(i,j,k) < 0) {
-        px(i,j,k) = dxi * (p(i+1,j,k) - phieb(i,j,k)) / (Real(1.0) - Real(2.0) * ecx(i,j,k));
+        Real h = Real(1.0) - Real(2.0) * ecx(i,j,k);
+        if (h == Real(0.0)) {
+            px(i,j,k) = Real(0.0);
+        } else {
+            px(i,j,k) = dxi * (p(i+1,j,k) - phieb(i,j,k)) / h;
+        }
```

Comment:
Potentially a good case for the ternary operator?
```suggestion
        px(i,j,k) = (h == 0_rt) ? 0_rt : dxi * (p(i+1,j,k) - phieb(i,j,k)) / h;
```

Gave me for some reason sometimes more efficient code. But either way is cool.